app/services/step_content_parser.rb

```diff
@@ -45,12 +45,24 @@ def base_paths(step_text)
     end
   end
 
+  def all_paths(step_text)
+    absolute_paths(step_text).map { |link| link[0..3] == "http" ? link : prefix_govuk(link) }
+  end
+
 private
 
   def relative_paths(content)
     content.scan(LINK_REGEX).select { |href| href[0] =~ /^\/[a-z0-9]+.*/i if href.any? }.flatten
   end
 
+  def absolute_paths(content)
+    content.scan(LINK_REGEX).flatten
```

Comment:
This method is misleading.  It doesn't just return absolute paths, but a mix of absolute and relative_paths.